packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/TeamPromptResponseSummaryCard.tsx

```diff
@@ -8,39 +8,43 @@ import {PALETTE} from '~/styles/paletteV3'
 import {createEditorExtensions} from '../../../../../components/promptResponse/tiptapConfig'
 
 const responseSummaryCardStyles: React.CSSProperties = {
-  margin: '12px',
-  width: '251px'
+  display: 'inherit',
```

Comment:
ultimately, this will be displayed by table and if it's a div, it'd break the layout a little